package.json

```diff
@@ -83,6 +87,7 @@
     "directory-tree-webpack-plugin": "0.3",
     "duplexer": "^0.1.1",
     "eslint": "4.19.1",
+    "eslint-config-prettier": "^4.2.0",
```

Comment:
Please pin exact version as you did with `prettier`

Reply:
Pinned